io.github.nyre221.kiview.json

```diff
@@ -0,0 +1,44 @@
+{
+   "id": "io.github.nyre221.kiview",
+   "runtime": "org.kde.Platform",
+   "runtime-version": "6.6",
+   "sdk": "org.kde.Sdk",
+   "base": "io.qt.qtwebengine.BaseApp",
+   "base-version": "6.6",
+   "command": "kiview",
+   "build-options": {
+   "env": [
+      "QTWEBENGINE_DISABLE_SANDBOX=1"
+    ]
+  },
+
```

Comment:
Why is sandbox disabled? It works fine for every app on Flathub.

Reply:
https://github.com/flathub/io.qt.qtwebengine.BaseApp

>
> During packaging of a Flatpak application there is no session bus socket in the sandbox, so if the build process needs running QtWebEngine, then it will fail.
> The workaround is to disable sandboxing like this:
>
> build-options:
>   env:
>     QTWEBENGINE_DISABLE_SANDBOX: '1'

I can remove it if it's not needed.

Reply:
No that's some old thing, it should work. I'll remove it.

Reply:
Also I don't see the app needing a running Qtwebengine process when building, you can drop it.

Reply:
removed